docs/use_cases/model_applications/s2s_mid_lat/UserScript_obsERA_obsOnly_Blocking.py

```diff
@@ -1,173 +1,209 @@
+#
+# The methodology in Miller & Wang (2019, 2022) first computes the Central Blocking Latitude 
+# (CBL) or storm track.  Allowing for an offset north and south of the storm track, reversals 
+# in geopotential height are then identified as Instantaneously Blocked longitudes (IBLs).  These 
+# IBLs are grouped when consective longitudes are blocked (GIBLs) and then blocks are identified 
+# by applying thresholds to ensure the large-scale, quasi-stationary characteristics of blocking 
+# anticyclones are met.
+#
+# This use case is a simplified version of the UserScript_fcstGFS_obs_ERA_Blocking use case.  While 
+# that use case evaluates blocking for both the model and observations, this case shows how to run 
+# the blocking calculation on observations only.  The setup is simpler and requires fewer MET runs 
+# than the UserScript_fcstGFS_obs_ERA_Blocking use case.  The original code for computing blocking 
+# came from Douglas Miller.
+#
+#  * Miller, D. E., and Z. Wang, 2019a: Skillful seasonal prediction of Eurasian winter blocking and extreme temperature frequency. Geophys. Res. Lett., 46, 11 530–11 538, https://doi.org/10.1029/2019GL085035.
+#  * Miller, D. E., and Z. Wang, 2022: Northern Hemisphere Winter Blocking: Differing Onset Mechanisms across regions. J. Atmos. Sci., 79, 1291-1309, https://doi.org/10.1175/JAS-D-21-0104.1.
+#  * Masato, G., B. J. Hoskins, and T. J. Woollings, 2013: Winter and summer Northern Hemisphere blocking in CMIP5 models. J. Climate, 26, 7044–7059, https://doi.org/10.1175/JCLI-D-12-00466.1.
+#  * Kitano, Y., and T. J. Yamada, 2016: Relationship between atmospheric blocking and cold day extremes in current and RCP8.5 future climate conditions over Japan and the surrounding area. Atmos. Sci. Lett., 17, 616–622, https://doi.org/10.1002/asl.711.
 
 ##############################################################################
-# Datasets
-# --------
+# Version Added
+# -------------
 #
-#  * Forecast dataset: None 
-#  * Observation dataset: ERA Reanlaysis 500 mb height.
+# METplus version 4.0.0
 
 ##############################################################################
-# External Dependencies
-# ---------------------
-#
-# You will need to use a version of Python 3.6+ that has the following packages installed::
-#
-# * numpy
-# * netCDF4
-# * datetime
-# * bisect
-# * scipy
+# Datasets
+# --------
 #
-# If the version of Python used to compile MET did not have these libraries at the time of compilation, you will need to add these packages or create a new Python environment with these packages.
+# **Forecast dataset:** None.
 #
-# If this is the case, you will need to set the MET_PYTHON_EXE environment variable to the path of the version of Python you want to use. If you want this version of Python to only apply to this use case, set it in the [user_env_vars] section of a METplus configuration file.:
+# **Observation dataset:** ERA Reanlaysis 500 mb height for DJF 1979 - 2017
```

Comment:
```suggestion
# **Observation:** ERA Reanlaysis 500 mb height for DJF 1979 - 2017
```
Matching the template. "datasets" is redundant since it's in the Datasets section.